### PR TITLE
feat: make createdBy and updatedBy columns not null, add default value

### PR DESCRIPTION
**Pull request by xixas**
Modified the database schema to make the 'createdBy' and 'updatedBy' columns not null, ensuring that every record includes this information.
Also, added default values for these columns in the environment configuration file.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -13,4 +13,7 @@ DB_NAME=
 SMTP_HOST=
 SMTP_PORT=
 SMTP_USERNAME=
-SMTP_PASSWORD=
\ No newline at end of file
+SMTP_PASSWORD=
+
+# TEMP
```
Comment (vishnu-kyatannawar) [inline .env.example:18]: What is temp?

File: .env.example [hunk 2]
```diff
@@ -13,4 +13,7 @@ DB_NAME=
...
+CREATED_BY=
```
Comment (vishnu-kyatannawar) [inline .env.example:19]: Can make it APP_NAME?

And add the comment saying to give the name of the app for which this is being used.

If pinestem then PINESTEM  will be value for this.

**Review by vishnu-kyatannawar — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (xixas) [reply .env.example:18]: For Temporary Values

File: .env.example (hunk 2 above)
Comment (xixas) [reply .env.example:19]: ok

Comment (xixas) [reply .env.example:19]: Updated

**Review by vishnu-kyatannawar — APPROVED** [review]

**Pull request merged**